src/main/java/com/denizenscript/depenizen/bukkit/events/jobs/JobsJobsExpGainScriptEvent.java

```diff
@@ -72,6 +76,8 @@ public ObjectTag getContext(String name) {
             case "job" -> job;
             case "experience" -> new ElementTag(event.getExp());
             case "action" -> new ElementTag(event.getActionInfo().getType().getName(), true);
+            case "entity" -> event.getLivingEntity() == null ? null : new EntityTag(event.getLivingEntity());
```

Comment:
Sorry for the delay, my laptop was out for 3 months so wasn't able to review PRs or anything sadly - assuming this entity could be a player/NPC, need to call `#getDenizenObject` on that `EntityTag` (same for the other event)

Reply:
Done :slightly_smiling_face: